src/test/java/edu/kpi/testcourse/property_tests/AddGetTests.java

```diff
@@ -0,0 +1,70 @@
+package edu.kpi.testcourse.property_tests;
+import edu.kpi.testcourse.dataservice.DataService;
+import edu.kpi.testcourse.dataservice.UrlAlias;
+import edu.kpi.testcourse.dataservice.User;
+import edu.kpi.testcourse.urlservice.AliasInfo;
+import edu.kpi.testcourse.urlservice.UrlService;
+import static org.quicktheories.generators.SourceDSL.strings;
+import io.micronaut.context.BeanContext;
+import io.micronaut.test.extensions.junit5.annotation.MicronautTest;
+import javax.inject.Inject;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import static org.quicktheories.QuickTheory.qt;
+
+@MicronautTest
+public class AddGetTests {
+  @Inject DataService dataService;
+  @Inject UrlService urlService;
+
+  @BeforeEach
+  void beforeEach() {
+    dataService.clear();
+    dataService.addUser(new User("jusovch", "j2202"));
```

Comment:
I hope it's not your reall password on github))0))0)

Reply:
just on my credit card)))